project/tychuang1211/README.md

```diff
@@ -0,0 +1,59 @@
+# Improve modmesh camera control 
+
+## Basic Information
+
+Implement a new QT camera controller class to improve current camera control of modmesh.
+
+GitHub repository: https://github.com/tychuang1211/modmesh/tree/70-improve-camera-control
+
+## Problem to Solve
+
+Make camera control more straight forward to use.
```

Comment:
Your description to the problem does not contain a lot of numerical calculations.  Could you please elaborate how it may related to numerics?